python/lsst/pipe/analysis/colorAnalysis.py

```diff
@@ -34,39 +35,40 @@ class ColorTransform(Config):
                             doc="Maximum values for colors so that this is useful")
 
     @classmethod
-    def fromValues(cls, description, plot, coeffs, requireGreater={}, requireLess={}):
+    def fromValues(cls, description, subDescription, plot, coeffs, requireGreater={}, requireLess={}):
```

Comment:
Instead of adding `subDescription` (which just feels tacky, especially since it gets set to `""` several times), why not just replace the `"Ivezic w perpendicular"` with `"griBlue"`? Like you say, people don't remember the identity of the wxy.